bindings/pyroot/pythonizations/test/helper.py

```diff
@@ -0,0 +1,219 @@
+import ROOT
```

Comment:
I don't think it's wise to have a separate module just for defining a bunch of functions. I would prefer if everything needed for your Filter tests was in the same module